admin/services/check-form.service.js

```diff
@@ -144,6 +144,7 @@ const checkFormService = {
    * @param CheckWindowsByForm
    * @returns {Promise.<void>}
    */
+  // WARN this expects a CheckWindow but is passed a check form in controllers/check-form.js
```

Comment:
@pris54 @GeorgeChatzigiannis @jon-shipley This method is called from `controllers/check-form.js` and expects a `checkWindow` as the first parameter.  It is passed a checkForm mongoose object.
Anyone remember this code or provide some clarity?